requirements.txt

```diff
@@ -0,0 +1,2 @@
+setuptools~=57.5.0
```

Comment:
setuptools is not a part of your application and should not be included in the list of project dependencies